database/seedData.js

```diff
@@ -1 +1,29 @@
-const faker = require('faker');
\ No newline at end of file
+const faker = require('faker');
+
+let makeData = (num) => {
+  let ratingGen = Math.floor(Math.random() * 5) + 1;
+  let listingGen = Math.floor(Math.random() * 200);
```

Comment:
Populating database with random id (per listing) may be tricky. Would it make more sense to cover everything from 0-199?